pysensu_yelp/__init__.py

```diff
@@ -162,8 +160,7 @@ def human_to_seconds(string):
                 seconds += value * interval_dict[unit]
                 string = string[match.end():]
             else:
-                raise Exception("'{0}' unit not present in {1}".format(
-                    unit, interval_dict.keys()))
+                raise Exception(f"'{unit}' unit not present in {interval_dict.keys()}")
```

Comment:
This is _probably_ wrong as formatting a iterator is probably not what is wanted here.

Reply:
Thanks for checking. Undid this change.